stable/etcd-operator/templates/backup-operator-deployment.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.deployments.backupOperator }}
 ---
-apiVersion: apps/v1beta1
+apiVersion: extensions/v1beta1
```

Comment:
For deployments can you please use `apps/v1` or `apps/v1beta2` for the `apiVersion` on all deployments. The extensions api version has long since been deprecated.

Reply:
oh right, i got it the wrong way. thanks @mattfarina